test/orm/test_scoping.py

```diff
@@ -111,11 +111,11 @@ def test_call_with_kwargs(self):
         Session = scoped_session(sa.orm.sessionmaker(), mock_scope_func)
 
         s0 = SessionMaker()
-        assert s0.autocommit == False
+        assert not s0.autocommit
```

Comment:
So!  yes this is good.  it would be better, if you are so inspired, to use SQLAlchemy's is_true and is_false assertions, can you force-push a new change here?   the imports are:

```
from sqlalchemy.testing import is_false
from sqlalchemy.testing import is_true

```
then:
```

is_false(s0.autocommit)
is_false(s1.autocommit)
is_true(s2.autocommit)
```

Reply:
Thank you for your suggestion. I'll push new change. :)